client/packages/reports/src/DetailView/Toolbar.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import {
+  AppBarContentPortal,
+  Grid,
+  BookIcon,
+  Typography,
+  useTranslation,
+  ChevronDownIcon,
+  LocaleKey,
+} from '@openmsupply-client/common';
+
+interface ToolbarProps {
+  reportName: string;
+}
+
+export const Toolbar = ({ reportName }: ToolbarProps) => {
+  const t = useTranslation();
+  const [expand, setExpand] = useState(false);
+  const chevronCommonStyles = {
+    width: '0.6em',
+    marginTop: '0.1em',
+    height: '0.6em',
+  };
+  const reportFormat = reportName.toLowerCase().replace(' ', '-');
+
+  return (
+    <AppBarContentPortal
+      sx={{
+        display: 'flex',
+        flex: 1,
+        marginBottom: 1,
+        flexDirection: 'column',
+      }}
+    >
+      <Grid width={800}>
+        <Grid display="flex" flexDirection="row">
+          <BookIcon
+            color="primary"
+            sx={{ width: '1em', marginTop: '0.1em', height: '0.7em' }}
+          />
+          <Typography
+            variant="body2"
+            alignItems="center"
+            display="flex"
+            sx={{
+              cursor: 'pointer',
+              color: 'secondary.main',
+            }}
+            onClick={() => setExpand(!expand)}
+          >
+            {t('messages.how-to-read-report', { reportName })}
+            {expand ? (
+              <ChevronDownIcon sx={{ ...chevronCommonStyles }} />
+            ) : (
+              <ChevronDownIcon
+                sx={{ transform: 'rotate(-90deg)', ...chevronCommonStyles }}
+              />
+            )}
+          </Typography>
+        </Grid>
+        {expand && (
+          <Grid sx={{ paddingLeft: 3 }}>
+            <Typography sx={{ textWrap: 'wrap' }} variant="body2">
+              {t(`messages.how-to-read-${reportFormat}` as LocaleKey)
+                .split('\n')
+                .map((line, index) => (
+                  <React.Fragment key={index}>
+                    {line}
+                    <br />
+                  </React.Fragment>
+                ))}
```

Comment:
Had to do this since \n wasn't inputting the text on new line... 👀 tho... it does input in a new line in console 👀

Reply:
did a quick google, looks like you can use CSS instead... add `whiteSpace: `pre-line` to the Typography component... maybe a little nicer?

Reply:
Oh the more you learn about CSS!